src/Tools/dotnet-monitor/dotnet-monitor.csproj

```diff
@@ -13,17 +13,6 @@
     <!-- This forces the creation of a checksum file and uploads it to blob storage
          using this name as part of the blob relative path. -->
     <BlobGroupPrefix>monitor</BlobGroupPrefix>
-    <!-- File locking warnings (that are typically resolved with automatic retries) are failing builds
-         quite frequently with messages such as:
-         
-         error MSB3026: (NETCORE_ENGINEERING_TELEMETRY=Build) Could not copy "...\dotnet-monitor.dll" to
-         "...\dotnet-monitor.dll". Beginning retry 1 in 1000ms. The process cannot access the file
```

Comment:
So moving pools addressed this issue I assume?

Reply:
No, this was fixed by the fact that the FunctionalTests project (which builds for netcoreapp3.1, net5.0, net6.0) now has a normally referenceable version of dotnet-monitor (respectively netcoreapp3.1, netcoreapp3.1, and net6.0) that doesn't need extra metadata for building the P2P reference. See the following the change that "fixes" this: https://github.com/dotnet/dotnet-monitor/pull/1098/files#diff-d28bf43afbb390eb0e8e6dbfbf8d3f0564284a0bbab0850763a7252a69820bf6